### ISSUE TITLE
The content of the "integrations" page is confusing

### ISSUE THREAD

**Issue by anoadragon453** [matrix-org/matrix.org#1858]
Link: https://matrix.org/ecosystem/integrations/

From the text on the page:

> Integrations allow you to bring content to and from third party platforms to make your conversations more productive… or more silly. You can see them as bridges, but not for chat.

The list here appears to be for application services or bots that bridge content (but not necessarily messages) from third-party platforms. That's fair, but then why are Hemppa, maubot and opsdroid - bot-building SDKs - on there?

And things like  is definitely an *integration*, but it doesn't bridge any content from a third-party platform. All content is generated entirely by Matrix users. And Matrix Registration Bot is definitely matrix-only.

I think the description of this page should be updated to be more generic. Describing integrations as "extensions and added features to your Matrix experience", or similar.

I think it'd be nice to reorganise this page a bit.

* Move the matrix-rss-bridge to the Bridges page (and arguably the matrix-trello-bot as well).
* Have an "Integrations" section for things like matrix-hookshot.
* Have a "Widgets" section for the primarily widget-based stuff like Matrix BarCamp and NeoBoard.
* Have a "Bots" section for primarily chat-based bots like Matrix Registration Bot, matrix-eno-bot and matrix-feeder.
* Move Hemppa, maubot and opsdroid to the ["bot SDKs" page](https://matrix.org/ecosystem/sdks/), and potentially provide a link behind the text "want to build your own integrations? See the list of available Matrix SDKs".

Finally, there's a nice list of ready-to-use maubot plugins/bots here, which could help populate this page: https://maubotplugins.mssj.me/

**Comment by HarHarLinks** [comment]
What do you think about #1828 ? Also see #1539.

**Comment by anoadragon453** [comment]
> What do you think about https://github.com/matrix-org/matrix.org/pull/1828 ?

Mmm, it's a bit wordy. Perhaps:

> Integrations enhance your chat experience by connecting to third-party platforms, automating tasks with bots, customizing the UI with widgets, and more!
